pkg/model/components/cilium.go

```diff
@@ -139,6 +139,10 @@ func (b *CiliumOptionsBuilder) BuildOptions(o *kops.Cluster) error {
 		c.EnableL7Proxy = fi.PtrTo(true)
 	}
 
+	if c.EnableLocalRedirectPolicy == nil {
+		c.EnableLocalRedirectPolicy = fi.PtrTo(false)
+	}
+
```

Comment:
Please remove these changes. Defaults are needed only when set to `true`. The generated output should not change.